samples/ImageList/src/controls/SearchField.tsx

```diff
@@ -22,14 +22,14 @@ const _styles = {
         fontSize: 16,
         padding: 5,
         margin: 10,
-        height: 30
+        height: 30,
     }),
 };
 
 export class SearchField extends RX.Component<SearchFieldProps, SearchFieldState> {
-    readonly state: SearchFieldState = { value: '' };
+    public readonly state: SearchFieldState = { value: '' };
```

Comment:
In TypeScript, "public" is the default modifier. For the coding style guideline we use in ReactXP, we don't include "public". The access modifier is included only if it's "protected" or "private". Please remove these public modifiers.